NSS-Day-Cohort-53/tabloid-full-stack-amicably-profound-insects
# Ap create tag

closes #43 
#### Changes Made
1. Added file `TagAddForm.js` to `components/tags` directory.
2. Modified file `TagRepository.cs` to include `addTag`.
3. Modified file `TagController.cs` to include `addTag` and functionality.
4. Modified file `tagManager.js` to include `addTag`.
5. Modified file `ApplicationViews.js` to include route path for TagAddForm.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ap-createTag
    ```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
6. Test app functionality.
    > When a user is logged in, the user should click on the Tag Management link in the navbar. They should see a list of tags, 
    >with an "add a new tag" button at the bottom. The user should click on the button, and see a field to add a new name for a 
    >tag. The user should click the "save" button to save their new tag name. They should be directed back to the list of tags, 
    >and see their new tag name in the alphabetical list of tags.
7. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
